### PR TITLE
#2809 added statistics service to core repo

### LINKED ISSUE #2809: Moving the statistics-service to Keptn API endpoint
Based on user requests, the statistics-service which is developed in keptn/sandbox should move closer to Keptn due to its useful features in providing a comprehensive usage statistic. Hence, it should become a part of Keptn.

**Tasks:**
Move microservice from sandbox to keptn/keptn (core)
 - Integration into the install process
 - Integration into build/test pipelines (push to DockerHub repo) 
 - Check development standards as applied for other services  
 - Expose the endpoint of the service on the Keptn API
 - Update of API documentation
 - Check resource footprint: Use monitoring data to verify the memory consumption of this service > update the resource allocation in the deployment manifest accordingly.
 - The CLI remains on *keptn/sandbox*

Definition of Done: 
- [ ] Service is part of the build/test pipeline
- [ ] Keptn API: `v1/statistics` exposes the endpoints of the statistics-service

### PR DESCRIPTION
**Pull request by bacherfl**
Closes #2809

The resource consumption has been investigated in detail here: [Resource consumption details](https://github.com/keptn-sandbox/statistics-service/pull/11)
Signed-off-by: Florian Bacher <florian.bacher@dynatrace.com>

### CODE REVIEW

File: statistics-service/skaffold.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+apiVersion: skaffold/v1beta13
+kind: Config
+build:
+  artifacts:
+    - image: keptnsandbox/statistics-service
```
Comment (christian-kreuzberger-dtx) [inline statistics-service/skaffold.yaml:6]: please change this to keptn/statistics-service

**Comment by christian-kreuzberger-dtx** [comment]
Do we need all those files in the `statistics-service/swagger-ui` subfolder?

**Review by christian-kreuzberger-dtx — CHANGES_REQUESTED** [review]
I've left some comments, please address tehm.

File: statistics-service/skaffold.yaml (hunk 1 above)
Comment (bacherfl) [reply statistics-service/skaffold.yaml]: done

**Comment by bacherfl** [comment]
> Do we need all those files in the `statistics-service/swagger-ui` subfolder?

afaik yes, those are needed for the swagger-ui to function properly

**Comment by bacherfl** [comment]
I something seemed to have gone wrong when merging the master branch into this one. I will create a new PR

**Pull request closed without merging**